Resources/Textures/Objects/Storage/happyhonk/clown.rsi/meta.json

```
@@ -0,0 +1,25 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Created by github: brainfood1183",
```

(same for the rest of copyrights)
```suggestion
  "copyright": "Made by  brainfood1183 (github)",
```